ContactMechanics/Systems.py

```diff
@@ -520,28 +520,32 @@ def minimize_proxy(self, solver=constrained_conjugate_gradients, **kwargs):
             self.substrate.check()
         return result
 
-
-    def primal_objective(self,offset,pot=False,gradient=True):
-        """
-        To solve the primal objective using gap as the variable.
+    def primal_objective(self, offset, pot=False, gradient=True):
+        r"""To solve the primal objective using gap as the variable.
         Can be fed directly to standard solvers ex: scipy solvers etc.
 
```

Comment:
"returns the elastic energy and it's gradient (negative of the forces) as a function of the gap."